nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -258,6 +267,9 @@ public String getName() {
     }
 
     public void set(String key, Object value) {
+        if (value instanceof Calendar || value instanceof Date) {
+            throw new IllegalArgumentException(MSG_DATE_UNSUPPORTED);
+        }
```

Comment:
Yes, good point. I didn't notice that. By the way, why do we have 2 methods which do the same thing, would it be a duplicate?